app/lib/forms/funding_your_npq.rb

```diff
@@ -26,11 +26,11 @@ def course
 
     def options
       [
-        (option("school", "My school or college is covering the cost", link_errors: true) if works_in_school?),
+        (option("school", "My workplace is covering the cost", link_errors: true) if works_in_school?),
         (option("trust", "My trust is paying") if works_in_school? && inside_catchment?),
         (option("employer", "My employer is paying") if !inside_catchment? || !works_in_school?),
```

Comment:
I couldn't see this option in the latest lucid flow, but it looked to me like we should keep it.

Reply:
That's a good point, would you be able to run that past James and see what he thinks?

Reply:
James mentioned we should check with Dom, but we both feel it's better to leave it for now and remove it later if we need to.